docs/configuration-reference.md

```diff
@@ -1524,19 +1524,43 @@ running on. It provides a significant amount of collectors that are responsible
 for monitoring various aspects of the host system.
 
 Note that if running the Agent in a container, you will need to bind mount
-folders from the host system so the integration can monitor them:
+folders from the host system so the integration can monitor them. You can use
+the example below, making sure to replace `/path/to/config.yaml` with a path on
+your host machine where an Agent configuration file is:
 
 ```
 docker run \
   --net="host" \
   --pid="host" \
   --cap-add=SYS_TIME \
-  -v "/:/host:ro,rslave" \
+  -v "/:/host/root:ro,rslave" \
+  -v "/dev:/host/dev:ro,rslave" \
+  -v "/proc:/host/proc:ro,rslave" \
   -v /tmp/agent:/etc/agent \
   -v /path/to/config.yaml:/etc/agent-config/agent.yaml \
   grafana/agent:v0.5.0 \
-  --config.file=/etc/agent-config/agent.yaml \
-  --integrations.node_exporter.rootfs-path=/host
+  --config.file=/etc/agent-config/agent.yaml
+```
+
+Use this configuration file for testing out `node_exporter` support, replacing
+the `prometheus_remote_write` settings with settings appropriate for you:
```

Comment:
There's no remote write stanza in the YAML below? Am I missing something?

Reply:
added, PTAL